platform/core/src/types/HangingProtocol.ts

```diff
@@ -106,35 +130,58 @@ type ViewportOptions = {
   customViewportProps?: Record<string, unknown>;
 };
 
-type DisplaySetOptions = {
+export type DisplaySetOptions = {
   // The id is used to choose which display set selector to apply here
   id: string;
   // An offset to allow display secondary series, for example
   // to display the second matching series (displaySetIndex==1)
   // This cannot easily be done with the matching rules directly.
   displaySetIndex?: number;
+
+  // A key to select a display set UID to reuse from the existing sets.
+  reuseId?: string;
+
   // The options to apply to the display set.
   options?: Record<string, unknown>;
 };
 
-type Viewport = {
+export type Viewport = {
   viewportOptions: ViewportOptions;
   displaySets: DisplaySetOptions[];
+  displaySetsByPosition?: Record<string, DisplaySetOptions[]>;
```

Comment:
I don't understand "One" viewport why should have a `displaySetsByPosition`? what does that even mean?

Reply:
Removed